feed-page/post/README.md

```diff
@@ -1 +1,57 @@
-# Post
\ No newline at end of file
+# Post
+
+```json
+{
+    "$type": "post@1", // Type definition with spec version number (1)
+    "id": 5, // Required, this id MUST be unique on the page this post is on. For example, this post could have the full id d448f1562c20dbafa42badd9f88560cd1adb2f177b30f0aa048cb243e55d37bd/feed/posts/1/5 (userId/structure/feedId/pageId/postId)
+
+    "repostOf":"5335179d69a3191ccb6329dcc0d2aaac2cada7ce145cbbd5543c8c2ee97e2a4a/feed/posts/0/11", // Optional, full ID of the post being reposted (If this key is present, this post is a repost and does not need to contain a "content")
+    "commentTo":"5335179d69a3191ccb6329dcc0d2aaac2cada7ce145cbbd5543c8c2ee97e2a4a/feed/posts/0/9", // Optional, full ID of the post this posts is commenting on
+    "parentHash":"sha256:24126b48452ce0243ae0f6bd3a3e0f9ee06ac01d5dab557c365b3e1bb8972fa5", // sha256sum of the JSON string of the post being reposted or commented on to prevent edits
+
+    "content": { // Every field in this map is OPTIONAL, if not specified otherwise
+        "mediaDuration": 1235312612, // duration of video or audio in milliseconds (int)
+        "video:": { // video content with different resolution and file formats
```

Comment:
What if we have a video and audio in the same post? Is it possible? Shouldn't `mediaDuration` be part of the "video" and "audio" object?